packages/webdriver-utils/src/driver.js

```diff
@@ -0,0 +1,51 @@
+import utils from '@percy/sdk-utils';
+import Cache from './util/cache.js';
+const { request } = utils;
+
+export default class Driver {
+  constructor(sessionId, executorUrl) {
+    this.sessionId = sessionId;
+    this.executorUrl = executorUrl.includes('@') ? `https://${executorUrl.split('@')[1]}` : executorUrl;
```

Comment:
Also can we add this check below this 😅 
```js
this.executorUrl = executorUrl.endsWith('/') ? this.executorUrl.slice(0, -1) : this.executorUrl;
```

It turns out that `C#` ends up adding `/` at the end which causes the request to fail so if we add this check then `C#` test passes.